app/routes/index.tsx

```diff
@@ -18,7 +20,8 @@ function IndexRoute() {
       <Spacer size="large" />
       <ProblemSolutionSection />
       <Spacer size="medium" />
-      <BlogSection articles={[]} />
+      {/*  TODO: replace fixtures */}
+      <BlogSection articles={articles as unknown as ArticleCardProps[]} />
```

Comment:
I've added the fixtures here, so I could see if things were still looking okay.

Reply:
That's fine. I'll work on this bit today.